Card Carousel

AlexA0595 opened the pull request:
if List length == 0 it will show the edit button. The button just goes back at the moment. it display the tradies with dummy data at the moment

AlexA0595 committed:
Card Carousel

if List length == 0 it will show the edit button. The button just goes back at the moment. it display the tradies with dummy data at the moment

AlexA0595 committed:
Card Carousel

if List length == 0 it will show the edit button. The button just goes back at the moment. it display the tradies with dummy data at the moment

veronicanashed commented on src/Pages/Login/ForgotPasswordform.tsx:
```diff
@@ -18,16 +19,24 @@ const animate = {
         delay: 0.16,
     },
 };
+const questions = [
+    "What is your mother's maiden name?",
+    "What is your favorite color?",
+    "What was the name of your first pet?",
+];
```
These are an enum in Account.ts:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/126940746/229483361-2521af02-8442-4ca1-b40e-a6f2fa34a03b.png">

AlexA0595 committed:
Updates

Updates to button. and ${request.status} instead of list.status for cards

AlexA0595 committed:
Updates

Updates to button. and ${request.status} instead of list.status for cards

vnashed committed:
Card Carousel PR

1. removed unused dependencies from from package.json (and attached new package-lock.json)
2. Moved all carousel related stuff to its own file (and calling the carousel from the summary, passing in callback function and request object as props
